github/pulls.go

```diff
@@ -248,19 +248,29 @@ type PullRequestMergeResult struct {
 	Message *string `json:"message,omitempty"`
 }
 
+type PullRequestOptions struct {
```

Comment:
Go lint would say `PullRequestOptions` is exported and should be documented `// PullRequestOptions ...`.